.github/workflows/test.yaml

```diff
@@ -13,51 +13,11 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: true
-
-      # Install Catlin linting tool
-      - name: Install linting tools
-        run: |
-          set -Eeu
-          GOBIN=/usr/local/bin/ go install github.com/openshift-pipelines/tektoncd-catalog/cmd/catalog-cd@latest
-          git clone https://github.com/tektoncd/catlin
-          cd catlin
-          make bin/catlin
-          sudo cp bin/catlin /usr/local/bin
-          catlin --help
-          catalog-cd --help
-
-      # run Catlin linting
-      - name: Run Catlin linting
-        run: |
-          set -Eeu
-          helm template task-oc . > task-oc.yaml
-          cat task-oc.yaml
-          catlin validate task-oc.yaml || true
-          helm template task-tkn . > task-tkn.yaml
-          cat task-tkn.yaml
-          catlin validate task-tkn.yaml || true
```

Comment:
Don't we require any kind of linting for the YAML files? @savitaashture

Reply:
same ans as abve